cardInfoAPI-Level1.yaml

```
@@ -0,0 +1,1402 @@
+
+    Correlation:
+      type: string
+      examples:
+        - f058ebd6-02f7-4d3f-942e-904344e8cde5
+      description: Schema for X-Correlation-ID header.
+
+    Date:
+      type: string
+      format: date
+      examples:
+        - 2018-04-13
+      description: General date that is used for different attributes, format is 'YYYY-MM-DD'.
+
+    DateTime:
+      title: Date-time
+      type: string
+      format: date-time
+      examples:
+        - 2018-04-13T11:11:11Z
+      description: General date-time that is used for different attributes, format is 'YYYY-MM-DDThh:mm:ssZ'.
+
+    LinksPagination:
+      description: Links (or cursors) returned in the answer of an API call.
+      type: object
+      properties:
+        self:
+          type: string
+          format: uri
+          description: Link / cursor to the current result set (based on the submitted pagination approach).
+          examples:
+            - /card-app/api/v1/products?offset=75&limit=25
+        first:
+          type: string
+          format: uri
+          description: Link / cursor to the first result set (based on the submitted pagination approach).
+          examples:
+            - /card-app/api/v1/products?offset=0&limit=25
+        previous:
+          type: string
+          format: uri
+          description: Link / cursor to the previous result set (based on the submitted pagination approach).
+          examples:
+            - /card-app/api/v1/products?offset=50&limit=25
+        next:
+          type: string
+          format: uri
+          description: Link / cursor to the next result set (based on the submitted pagination approach).
+          examples:
+            - /card-app/api/v1/products?offset=100&limit=25
+        last:
+          type: string
+          format: uri
+          description: Link / cursor to the last result set (based on the submitted pagination approach).
+          examples:
+            - /card-app/api/v1/products?offset=150&limit=25
+
+    CommonErrorResponse:
+      title: Common Error Response
+      description: General error response structure used in different response codes.
+      type: object
+      properties:
+        type:
+          $ref: '#/components/schemas/CommonErrorType'
+        title:
+          type: string
+          description: Short name for the problem.
+          examples:
+            - This is the general problem description
+        detail:
+          type: string
+          description: More details about the problem.
+          examples:
+            - Detailed problem description with respect to the current request
+        instance:
+          type: string
+          description: Reference to an individual entity or request which caused the problem.
+          examples:
+            - path/to/corresponding/resource
+
+    CommonErrorType:
+      title: Common Error Type
+      description: Error Types for CommonErrorResponse.
+      type: string
+      enum:
```

I'm expecting only the related types per Error code. e.g. for HTTP 400:
```suggestion
      enum:
      - /problems/bad-request
```

all other can lead to misunderstanding.

---

We will replace the enums with a generic string "about:blank" to match the RFC 7807 definition which requires a URI.